game.py

```diff
@@ -151,6 +151,7 @@ def update():
                 current_level = (current_level + 1) % len(levels)
                 restart()
 
+        draw_text("Level: {}".format(current_level + 1), (0, 0))
```

Comment:
Format string maybe?

Reply:
This was really just a test, but sure.

Reply:
I know how you feel! But this example code, and if we can teach young programmers about format strings we all win.

Reply:
One downside of format strings is that they are a fairly new addition. I'm not sure if there are any mainstream linuxes *cough* centos that may not have support for them yet